src/App/GeoFeaturePyImp.cpp

```diff
@@ -57,6 +57,27 @@ PyObject* GeoFeaturePy::getGlobalPlacement(PyObject * args) {
     }
 }
 
+PyObject* GeoFeaturePy::getGlobalPlacementOf(PyObject * args) {
+
+    PyObject* pyTargetObj;
+    PyObject* pyRootObj;
```

Comment:
(GitHub won't let me commit this change myself because the branch is in Ondsel's repo)
```suggestion
    PyObject* pyTargetObj {nullptr};
    PyObject* pyRootObj {nullptr};
```